.env.example

```
@@ -1,92 +1,57 @@
+# Shizzle configuration template. Copy to a NEW gitignored .env for Compose.
+# Fill placeholders for the target environment; never commit real secrets.
+# Native uv/uvicorn processes read process environment, not this file automatically.
+# See docs/SETUP.md and deploy/runpod/README.md.
 
-# ── App ──────────────────────────────────────────────────────────────────────
-# [api] Shared passcode friends enter once per device. Rotating it (and bumping
-# AUTH_VERSION) revokes every issued device token.
+# API authentication. An EMPTY passcode disables auth; require a real production value.
 SHIZZLE_PASSCODE=changeme
+TOKEN_SIGNING_SECRET=replace-with-a-long-random-secret
 AUTH_VERSION=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_VERSION key should go before the SHIZZLE_PASSCODE key

(UnorderedKey)